.github/workflows/devnet-deploys.yml

```diff
@@ -97,8 +97,8 @@ jobs:
       - name: Deploy L1 Contracts
         if: steps.check_changes_release.outputs.result == 'true'
         run: |
-          docker pull aztecprotocol/cli:${{ env.DEPLOY_TAG }}
-          docker run aztecprotocol/cli:${{ env.DEPLOY_TAG }} \
+          docker pull aztecprotocol/aztec:${{ env.DEPLOY_TAG }}
```

Comment:
Nice, will also need to remove "Build & Push cli image" step in the build job

Reply:
Done!